package.json

```diff
@@ -54,6 +54,8 @@
     "vip-wp": "dist/bin/vip-wp.js"
   },
   "scripts": {
+    "typescript:codegen:install-dependencies": "npm install --no-save @graphql-codegen/cli @graphql-codegen/typescript @graphql-codegen/near-operation-file-preset @graphql-codegen/typescript-operations",
```

Comment:
What do you think of this? I do prefer adding the dev dependency to the project itself, but perhaps we don't need this too often.

We could also create a mini sub-project just to generate the types if that's preferred.

Reply:
Whatever works. I am OK with either way.